### PR TITLE
#10869: Distributing reviews required for ops

### PR DESCRIPTION
**Pull request by eyonland**
### Ticket
Link to Github Issue

### Problem description
Provide context for the problem.

### What's changed
Describe the approach used to solve the problem.
Summarize the changes made and its impact.

### Checklist
- [ ] Post commit CI passes
- [ ] Model regression CI testing passes (if applicable)
- [ ] New/Existing tests provide coverage for changes

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -114,10 +114,19 @@ ttnn/ @eyonland @arakhmati @cfjchu @xanderchin @TT-BrianLiu @ayerofieiev-tt @dma
 ttnn/**/kernels/ # Removes the owners above from owning kernels unless specified afterwards
 ttnn/setup.py @tt-rkim
 ttnn/CMakeLists.txt @tt-rkim @ayerofieiev-tt
+ttnn/cpp/ttnn/operations/ccl/ @SeanNijjar @TT-BrianLiu
+ttnn/cpp/ttnn/operations/reduce_scatter/ @SeanNijjar @TT-BrianLiu
+ttnn/cpp/ttnn/operations/pool/ @mywoodstock @nsmithtt
```
Comment (mywoodstock) [inline CODEOWNERS:119]: please replace @nsmithtt with @sankarmanoj-tt

**Review by arakhmati — APPROVED** [review]

File: CODEOWNERS [hunk 2]
```diff
@@ -114,10 +114,19 @@ ttnn/ @eyonland @arakhmati @cfjchu @xanderchin @TT-BrianLiu @ayerofieiev-tt @dma
...
+ttnn/cpp/ttnn/operations/upsample/ @mywoodstock @nsmithtt
```
Comment (mywoodstock) [inline CODEOWNERS:120]: please replace @nsmithtt with @shwetankTT

File: CODEOWNERS [hunk 3]
```diff
@@ -114,10 +114,19 @@ ttnn/ @eyonland @arakhmati @cfjchu @xanderchin @TT-BrianLiu @ayerofieiev-tt @dma
...
+ttnn/cpp/ttnn/operations/conv2d/ @mywoodstock @nsmithtt
```
Comment (mywoodstock) [inline CODEOWNERS:121]: please replace @nsmithtt with @pavlejosipovic

File: CODEOWNERS (hunk 2 above)
Comment (eyonland) [reply CODEOWNERS]: I did add @pavlejosipovic.   I'd like to start with Tenstorrent employees and see how it goes.

**Review by ayerofieiev-tt — APPROVED** [review]
@SeanNijjar @tarafdarTT @mywoodstock @yugaoTT @pavlejosipovic @sjameelTT we entrust you with a lot of power here. Please carry it responsibly. Keep the bar HIGH 🚀

File: CODEOWNERS (hunk 3 above)
Comment (eyonland) [reply CODEOWNERS]: I had left @nsmithtt as just a backup in case both you and @pavlepopovic are on vacation.

**Review by TT-BrianLiu — APPROVED** [review]

**Review by mywoodstock — APPROVED** [review]

File: CODEOWNERS (hunk 3 above)
Comment (eyonland) [reply CODEOWNERS]: Removing @nsmithtt

**Comment by SeanNijjar** [comment]
I humbly welcome @TT-BrianLiu as an honorary ccl developer

**Review by yugaoTT — APPROVED** [review]

File: CODEOWNERS [hunk 4]
```diff
@@ -114,10 +114,17 @@ ttnn/ @eyonland @arakhmati @cfjchu @xanderchin @TT-BrianLiu @ayerofieiev-tt @dma
...
 ttnn/CMakeLists.txt @tt-rkim @ayerofieiev-tt
+ttnn/cpp/ttnn/operations/ccl/ @SeanNijjar @TT-BrianLiu
+ttnn/cpp/ttnn/operations/pool/ @mywoodstock @pavlejosipovic
+ttnn/cpp/ttnn/operations/conv2d/ @mywoodstock @pavlejosipovic
+ttnn/cpp/ttnn/operations/data_movement/ @tarafdarTT @sjameelTT @yan-zaretskiy
 ttnn/cpp/ttnn/operations/matmul/ @TT-BrianLiu @bbradelTT @yugaoTT
+ttnn/cpp/ttnn/operations/eltwise/ @arakhmati @eyonland
+ttnn/cpp/ttnn/operations/reduction/ @SeanNijjar @tarafdarTT @sjameelTT
+ttnn/cpp/ttnn/operations/normalization/ @yugaoTT @tt-aho
+ttnn/cpp/ttnn/operations/embedding/ @tt-aho @TT-BrianLiu
```
Comment (tarafdarTT) [inline CODEOWNERS:125]: can you add me to embedding (@tarafdarTT)

**Review by tt-rkim — APPROVED** [review]

File: CODEOWNERS (hunk 2 above)
Comment (sankarmanoj-tt) [reply CODEOWNERS]: @shwetankTT  and I are Tenstorrent employees.

**Review by tt-aho — APPROVED** [review]

**Review by nsmithtt — APPROVED** [review]

File: CODEOWNERS (hunk 4 above)
Comment (eyonland) [reply CODEOWNERS]: done

File: CODEOWNERS [hunk 5]
```diff
@@ -114,10 +114,17 @@ ttnn/ @eyonland @arakhmati @cfjchu @xanderchin @TT-BrianLiu @ayerofieiev-tt @dma
 ttnn/**/kernels/ # Removes the owners above from owning kernels unless specified afterwards
 ttnn/setup.py @tt-rkim
 ttnn/CMakeLists.txt @tt-rkim @ayerofieiev-tt
+ttnn/cpp/ttnn/operations/ccl/ @SeanNijjar @TT-BrianLiu
+ttnn/cpp/ttnn/operations/pool/ @mywoodstock @pavlejosipovic
+ttnn/cpp/ttnn/operations/conv2d/ @mywoodstock @pavlejosipovic
```
Comment (mywoodstock) [inline CODEOWNERS:119]: @sankarmanoj-tt will also need to be added here. He is also part of the conv team.

Comment (tt-rkim) [reply CODEOWNERS:119]: Make the change and propose, this is already merged

**Pull request merged**